### ISSUE TITLE
[preview] preview can initially lag

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#6868]
It's unfortunately common for the first preview to lag by a second or two. The reason for this is that in order to show the preview, the debugger first has to compute the mapped expression. The mapping is performed by the parser worker, which in the case of original files is busy computing the original and generated ast scopes.

**Comment by darkwing** [comment]
Yeah, that can take a while.  Might be nice to have a message somewhere just so the user knows work is happening and the info will be shown shortly.

**Comment by jasonLaster** [comment]
We have a loading message in the scopes pane, but could do more there for sure.
